src/Calypso-SystemTools-Core/SycOpenDebuggingMenuCommand.extension.st

```diff
@@ -0,0 +1,15 @@
+Extension { #name : #SycOpenDebuggingMenuCommand }
+
+{ #category : #'*Calypso-SystemTools-Core' }
+SycOpenDebuggingMenuCommand class >> methodEditorShortcutActivation [
+	<classAnnotation>
+	
+	^CmdShortcutActivation by: $t meta for: ClySourceCodeContext
```

Comment:
It overrides "Source code" menu: SycOpenSourceCodeMenuCommand class>>#methodEditorShortcutActivation

Reply:
Ah yes! Thank you!
But now I need to find an available shortcut :/
Is there a way to know which is free?

Reply:
It will give you Commander based shortcuts:
```Smalltalk
CmdShortcutActivation registeredInstances groupedBy: #keyCombination
```
And try to select only commands for ClySourceCodeContext.
But there are also some old style text editor shortcuts. I don't know how to see them.

Reply:
We could use meta+d, the only user is ClyDebugTestCommand. 
What is this command for?

Reply:
I guess it is to debug a test :)

Reply:
But where is it used?
If there was an example I could see if there are possible conflicts.

Reply:
It is accessible on selected test method

Reply:
Ok, to me there is no conflict, unless we are supposed to be able to open submenus from the methods list?